enclave_mgmt/data_collection.py

```diff
@@ -0,0 +1,316 @@
+import json
```

Comment:
It occurred to me that the naming of this file is inconsistent in this repo which is currently `{verb}_{noun}`. What do you think about calling this `collect_data.py` instead?